bundle/DependencyInjection/Configuration.php

```diff
@@ -73,4 +75,94 @@ private function addAsynchronousIndexingSection(ArrayNodeDefinition $nodeDefinit
                 ->end()
             ->end();
     }
+
+    private function addPageIndexingSection(ArrayNodeDefinition $nodeDefinition): void
+    {
+        $keyValidator = static function ($v) {
+            foreach (array_keys($v) as $key) {
+                if (!is_string($key)) {
+                    return true;
+                }
+            }
+            return false;
+        };
+        $nodeDefinition
+            ->children()
+                ->arrayNode('page_indexing')
+                ->addDefaultsIfNotSet()
+                ->info('Page indexing configuration')
+                ->children()
+                    ->booleanNode('enabled')
+                        ->info('Use layouts page text indexing')
```

Comment:
"Use page indexing".

Reply:
fixed in 9e2818fbb70808cd454e96b7ac239d701d99bd7a